docs: update instructions for running rust examples (#1511)
n0-computer/iroh: iroh/README.md

## Running Examples

> Examples are located in `iroh/examples`. Run them with `cargo run --example`. eg: `cargo run --example hello-world`. Each example has a description on how to run in a comment at the top of the example file.

Reviewer: "has a description on how to run in a comment" 
```suggestion
Examples are located in `iroh/examples`. Run them with `cargo run --example`. eg: `cargo run --example hello-world`. At the top of each example file is a comment describing how to run the example.
```
